File: PRAK205-2410817210003-MUHAMMAD_MAULANA_AZHARI.c
#include <stdio.h>
#include <math.h>

int main(){
    float tinggi, hipotenusa = 0;
    scanf("%f", &tinggi); 
    scanf("%f", &hipotenusa); 
    float alas = sqrt((hipotenusa * hipotenusa) - (tinggi * tinggi));
    printf("Alas =  %.0f cm \n", alas);
    printf("Tinggi =  %.0f cm \n", tinggi);
    printf("Keliling =  %.0f cm \n", (alas + tinggi + hipotenusa));
    printf("Luas =  %.0f cm^2 \n", (0.5 * alas * tinggi));
    return 0;
}

File: PRAK205-2410817210003-MUHAMMAD_MAULANA_AZHARI.py
tinggi, hipotenusa = map(int, input().split())
alas = ((hipotenusa ** 2) - (tinggi ** 2)) ** 0.5
print("Alas = %d cm" %alas)
print("Tinggi = %d cm" %(tinggi))
print("Keliling = %d cm" %(alas + tinggi + hipotenusa))
print("Luas = %d cm^2" %(0.5 * alas * tinggi))

tinggi = int(input())
hipotenusa = int(input())
alas = ((hipotenusa ** 2) - (tinggi ** 2)) ** 0.5
print("Alas = %d cm" %alas)
print("Tinggi = %d cm" %(tinggi))
print("Keliling = %d cm" %(alas + tinggi + hipotenusa))
print("Luas = %d cm^2" %(0.5 * alas * tinggi))